Podfile

```diff
@@ -2,7 +2,7 @@
 use_frameworks!
 
 target 'BuildaUtils' do
-	pod 'SwiftSafe', '~> 0.1'
+	pod 'SwiftSafe', :git => 'https://github.com/Sajjon/SwiftSafe.git', :commit => '243f20e'
```

Comment:
I wonder if it would be possible to request an «official» release of the `Swift 3` support.

Would like to point to that rather than to a commit hash if at all possible.